src/main/java/com/es/phoneshop/model/product/Product.java

```diff
@@ -14,18 +15,26 @@
     private Currency currency;
     private int stock;
     private String imageUrl;
+    private ArrayList<DateAndPriceBean> priceHistory;
```

Comment:
https://medium.com/@sanchezdale/oop-good-practices-coding-to-the-interface-baea84fd60d3